### PR TITLE
Feature/df 394

### PR DESCRIPTION
**Pull request by mccallry**

### CODE REVIEW

File: src/data/datetime/unix.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+use wasm_bindgen::prelude::*;
+use crate::utils::seeder;
+
+#[wasm_bindgen]
+pub fn unix() -> String {
+  let (secs, nanos) = unix_gen();
+  concat_string!(secs.to_string(), nanos.to_string())
+}
```
Comment (clearfeld) [inline src/data/datetime/unix.rs]: I'd prefer this to have a different js_name than simply unix.

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ console.log(faking.f64());
 ### Unsupported Seeded Generation
 The following data generators do not support seeded generation:
 
-- [UUIDs](src/data/defaults/uuids.rs), any.
+- [UUIDs](src/data/defaults/uuids.rs), ([v4](src/data/defaults/uuids.rs#L28) | [v7](src/data/defaults/uuids.rs#L48)).
 - DateTime: Naive Date ([Before Today](src/data/datetime/date_naive.rs#L57) | [After Today](src/data/datetime/date_naive.rs#L83)).
 
 NOTE: uuids is currently the only data generator category that doesn't yet support seeded generation, expect results to differ for that one case.
```
Comment (clearfeld) [inline README.md]: We should get rid of this line.

File: README.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ console.log(faking.f64());
 ### Unsupported Seeded Generation
 The following data generators do not support seeded generation:
 
-- [UUIDs](src/data/defaults/uuids.rs), any.
+- [UUIDs](src/data/defaults/uuids.rs), ([v4](src/data/defaults/uuids.rs#L28) | [v7](src/data/defaults/uuids.rs#L48)).
```
Comment (clearfeld) [inline README.md]: v4 and v7 should be possible to made consistent using the uuid builder 
https://docs.rs/uuid/latest/src/uuid/builder.rs.html#583-587
https://docs.rs/uuid/latest/src/uuid/v7.rs.html#47-53

**Review by clearfeld — CHANGES_REQUESTED** [review]
Above

File: README.md (hunk 3 above)
Comment (mccallry) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (mccallry) [reply README.md]: Fixed

File: src/data/datetime/unix.rs (hunk 1 above)
Comment (mccallry) [reply src/data/datetime/unix.rs]: unix -> unix_ts

**Review by clearfeld — APPROVED** [review]

**Pull request merged**